src/v/config/configuration.cc

```diff
@@ -3697,6 +3697,16 @@ configuration::configuration()
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       std::chrono::milliseconds(1min),
       {.min = 10ms})
+  , iceberg_catalog_commit_interval_ms(
+      *this,
+      "iceberg_catalog_commit_interval_ms",
+      "How often the Iceberg coordinator commits topic files to the catalog. "
+      "This is the interval between commit transactions across all topics "
+      "monitored by the coordinator, not the interval between individual "
+      "commits.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      std::chrono::milliseconds(5ms),
+      {.min = 5ms /* I guess? idk. */})
```

Comment:
I think the final values will be TBD anyway, but maybe just copy the default/min from translation interval? (1min/10ms)

Reply:
cool sg